src/_data/resources.json

```diff
@@ -26,6 +26,11 @@
 			"description": "24 days of digital accessibility gifts during the season of giving and sharing.",
 			"url": "https://www.24a11y.com/"
 		},
+		{
+			"title": "Deque Blog",
```

Comment:
Can you move this into alphabetical order? It would be after "Blog - Karl Groves", starting on line 69.